src/collective/siteimprove/browser/utils.py

```diff
@@ -12,3 +15,23 @@ class AnonCanView(BrowserView):
     def __call__(self):
         """Can we see this"""
         return 'Anonymous' in rolesForPermissionOn('View', self.context)
+class UseDomainView(BrowserView):
+    """Returns true if the domain function is to be used in the siteimprove js header
+       false implies that input function will be used instead"""
+
+    def __call__(self):
+        use_domain = False # use input function by default
+
+        # figure out if domain function should be used instead
+        portal_state = getMultiAdapter((self.context, self.request), name=u'plone_portal_state')
+        site = portal_state.portal() #site root
+        site_path = site.getPhysicalPath();
+        context_path = self.context.getPhysicalPath()
+
+        # XXX This is not correct -- fix method to calculate when we are at root of site
```

Comment:
@alecpm 
How do we calculate the root of the site?
Excuse the ignorance but I see that when I'm at the "root" of the site. I see:
Pdb) self.context
<Document at /Plone/front-page>

IPloneSiteRoot.providedBy(self.context) returns False